qa-covid19.planx-pla.net/manifest.json

```
@@ -6,26 +6,28 @@
   "versions": {
     "arborist": "quay.io/cdis/arborist:2020.03",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
+    "covid19-etl": "quay.io/cdis/covid19-etl:1.0.0",
     "fence": "quay.io/cdis/fence:2020.03",
     "indexd": "quay.io/cdis/indexd:2020.03",
     "peregrine": "quay.io/cdis/peregrine:2020.03",
     "pidgin": "quay.io/cdis/pidgin:2020.03",
     "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
     "sheepdog": "quay.io/cdis/sheepdog:4.0.0",
-    "portal": "quay.io/cdis/data-portal:2.24.8",
+    "portal": "quay.io/cdis/data-portal:wip_covid19dashboard",
     "tube": "quay.io/cdis/tube:2020.03",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
     "spark": "quay.io/cdis/gen3-spark:2020.03",
     "hatchery": "quay.io/cdis/hatchery:2020.03",
     "wts": "quay.io/cdis/workspace-token-service:2020.03",
     "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.03",
-    "ambassador": "quay.io/datawire/ambassador:0.60.3"
+    "ambassador": "quay.io/datawire/ambassador:0.60.3",
+    "guppy": "quay.io/cdis/guppy:2020.03"
```

```suggestion
    "guppy": "quay.io/cdis/guppy:0.4.2"
```
to match what we're trying to get into prod: https://github.com/uc-cdis/cdis-manifest/pull/1317/files